### ISSUE TITLE
add version number to oranda css

### ISSUE THREAD

**Issue by ashleygwilliams** [axodotdev/oranda#212]
since #200 we're using local oranda css which is excellent, but we've dropped the idea of versions for the moment and hardcoded 0.0.0. we should... not do this. #200 is getting huge tho so i'll tackle it in another PR to help unblock @SaraVieira building more themes.
